cli/tests/unit/sleep_test.ts

```diff
@@ -0,0 +1,58 @@
+import { assert, unitTest } from "./test_util.ts";
+
+unitTest({ perms: { hrtime: true } }, function sleepSync(): void {
+  let start = performance.now();
+  Deno.sleepSync(0);
+  let after = performance.now();
+  assert(after - start >= 0);
```

Comment:
Does this actually test anything? I guess `sleepSync(0)` is a good smoke test, but I would remove this assert